### DISCUSSION TITLE
Treiber für ST7789 einbinden für den Wall E

### DISCUSSION THREAD

**Discussion by Mike2404-maker** [olikraus/u8g2#2537] [General]
Hallo Oliver,
ich wollte fragen ob es möglich den Treiber für ST7789  in die U8g2 Lib einzubinden, die meisten Displays in deutschland zum kaufen haben SPI Schnittstelle, leider reichen meine Programmierkenntnisse nicht aus um das selbst zu erledigen. Nun wollte ich dich fragen ob du helfen kannst`? Ich habe mir ein 1.4 Zoll Display mit Auflösung 240x240 gekauft, es ist mit 4 Pins angeschlossen hat aber 7 verfügbare
Pins. Es wäre super wenn du mir helfen könntest., Vielen Dank schon mal im vorraus für deine Mühe.

**Comment by olikraus** [comment]
U8g2 is spezialisiert auf monochrome (schwarz/weiß) displays.
ST7789 ist ein Farbdisplay. Zwar könnte u8g2 theoretisch das ST7789 unterstützen, aber es wäre dann eben nur schwarz-weiß und sehr langsam.
